web/src/lib/components/shared-components/show-shortcuts.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+  import Close from 'svelte-material-icons/Close.svelte';
+  import CircleIconButton from '../elements/buttons/circle-icon-button.svelte';
+
+  export let showShortcuts: boolean;
+  const shortcuts = {
+    general: [
+      { key: ['←', '→'], action: 'Previous or next photo' },
+      { key: ['Esc'], action: 'Back, close, or deselect' },
+      { key: ['/'], action: 'Search your photos' },
+    ],
+    actions: [
+      { key: ['f'], action: 'Favorite or unfavorite photo' },
+      { key: ['i'], action: 'Show or hide info' },
+      { key: ['⇧', 'a'], action: 'Archive or unarchive photo' },
+      { key: ['⇧', 'd'], action: 'Download' },
+      { key: ['Space'], action: 'Play or pause video' },
+      { key: ['Del'], action: 'Delete Asset' },
+    ],
+  };
+</script>
+
+<div class="absolute w-full h-full z-[99999]">
+  <div class="flex h-full w-full place-content-center place-items-center overflow-hidden bg-black/50">
+    <div
+      class="w-[650px] max-w-[125vw] rounded-3xl border bg-immich-bg shadow-sm dark:border-immich-dark-gray dark:bg-immich-dark-gray dark:text-immich-dark-fg"
+    >
+      <div class="relative px-4 pt-4">
+        <h1 class="font-medium text-immich-primary dark:text-immich-dark-primary px-4 py-4">Keyboard Shortcuts</h1>
+        <div class="absolute inset-y-0 right-0 px-4 py-4">
+          <CircleIconButton logo={Close} on:click={() => (showShortcuts = !showShortcuts)} />
```

Comment:
It would be better to just emit an event here 'close' and keep the boolean state of open or close in the parent component.